README.md

```
@@ -1,7 +1,7 @@
 # LANDFIRE Reference Conditions
 
-This repository contains R scripts to support extracting information from the LANDFIRE [Biophysical Settings (BpS) models](https://landfire.gov/bps-models.php) and model descriptions.
+This repository contains R scripts to support extracting information from the LANDFIRE [Biophysical Settings (BpS) models](https://landfire.gov/vegetation/bps-models) and model descriptions.
 
-The script **1_GenerateClassCrosswalk.R** creates a .csv table with [succession class](https://landfire.gov/sclass.php) information (*i.e.*, class, cover type, structural stage, maximum tree size class, and description) based on the LANDFIRE reference condition model descriptions. The script takes as input a folder with Word documents describing each BpS model. Files are available for direct download [here](https://www.landfire.gov/zip/LANDFIRE_CONUS_HI_BpS_Descriptions_May2022.zip).
+The script **1_GenerateClassCrosswalk.R** creates a .csv table with [succession class](https://landfire.gov/vegetation/sclass) information (*i.e.*, class, cover type, structural stage, maximum tree size class, and description) based on the LANDFIRE reference condition model descriptions. The script takes as input a folder with Word documents describing each BpS model. Files are available for direct download [here](https://landfire.gov/data-downloads/zip/LANDFIRE_CONUS-HI_BpS_Descriptions_Jan2023.zip).
```

markdownlint-cli2 (0.22.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)